packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/shippingInformation.gql.js

```diff
@@ -12,9 +12,38 @@ export const GET_SHIPPING_INFORMATION = gql`
     ${ShippingInformationFragment}
 `;
 
+export const GET_DEFAULT_SHIPPING = gql`
+    query GetDefaultShipping {
+        customer {
+            id
+            default_shipping
+        }
+    }
+`;
+
+export const SET_CUSTOMER_ADDRESS_ON_CART = gql`
+    mutation SetCustomerAddressOnCart($cartId: String!, $addressId: Int!) {
+        setShippingAddressesOnCart(
+            input: {
+                cart_id: $cartId
+                shipping_addresses: [{ customer_address_id: $addressId }]
+            }
+        ) @connection(key: "setShippingAddressesOnCart") {
+            cart {
+                id
+                ...ShippingInformationFragment
+            }
+        }
+    }
+    ${ShippingInformationFragment}
+`;
+
 export default {
-    mutations: {},
+    mutations: {
+        setDefaultAddressMutation: SET_CUSTOMER_ADDRESS_ON_CART
```

Comment:
You changed the definition name but not the export - I still think could be clarified by renaming it away from `default` but I'll leave it up to you.

Reply:
In the scope of `shippingInformation`, this mutation is only ever used to set the default address on the cart, so I've renamed to `setDefaultAddressOnCartMutation` in defaf9f.